app/pracuj.py

```diff
@@ -0,0 +1,142 @@
+from bs4 import BeautifulSoup
+from selenium import webdriver
+from selenium.webdriver.chrome.service import Service
+
+import pandas as pd
+import re
+import os
+
+
+def pracuj_switch(url):
+
```

Comment:
Ta funkcja powinna się inaczej nazywać. Switch odnosi się raczej do przełączania on-off niż przechodzenia na kolejne podstrony. Bez analizy kodu nie byłem w stanie powiedzieć co robi ta funkcja.

Plus to, że pewne jej części można wyciągnąć do osobnej funkcji, przede wszystkim tworzenie drivera. To w ogóle może być to `commons.py` bo dotyczy też JJIT.

Goły except raczej nie powinien występować tak jak tutaj.